sentry/src/main/java/io/sentry/util/SampleRateUtil.java

```diff
@@ -15,7 +15,7 @@ public static boolean isValidSampleRate(@Nullable Double sampleRate, boolean all
       return allowNull;
     }
 
-    return !(sampleRate.isNaN() || (sampleRate > 1.0 || sampleRate <= 0.0));
+    return !(sampleRate.isNaN() || sampleRate > 1.0 || sampleRate <= 0.0);
```

Comment:
Apparently a new private method that takes a Double and do this check would avoid code duplication, See this method and `isValidTracesSampleRate`.

Reply:
may i rename `isValidTracesSampleRate` to `isValidTracesOrProfilesSampleRate` and use just one function, as they do basically the same thing?

Reply:
`isValidSampleRate` should be enough IMO, as long as both functions do the same.

Reply:
yeah, but `isValidSampleRate` already exists and does the same check, expect the rate cannot be 0.0

Reply:
Then your call.